web/packages/teleport/src/Roles/RoleEditor/StandardEditor.tsx

```diff
@@ -896,6 +927,84 @@ export function WindowsDesktopAccessSpecSection({
   );
 }
 
+export function AdminRules({
+  value,
+  isProcessing,
+  validation,
+  onChange,
+}: SectionProps<RuleModel[], AdminRuleValidationResult[]>) {
+  function addRule() {
+    onChange?.([...value, newRuleModel()]);
+  }
+  function setRule(rule: RuleModel) {
+    onChange?.(value.map(r => (r.id === rule.id ? rule : r)));
+  }
+  function removeRule(id: string) {
+    onChange?.(value.filter(r => r.id !== id));
+  }
+  return (
+    <Flex flexDirection="column" gap={3}>
+      {value.map((rule, i) => (
+        <AdminRule
+          key={rule.id}
+          isProcessing={isProcessing}
+          value={rule}
+          onChange={setRule}
+          validation={validation[i]}
+          onRemove={() => removeRule(rule.id)}
+        />
+      ))}
+      <ButtonSecondary alignSelf="start" onClick={addRule}>
+        <Icon.Plus size="small" mr={2} />
+        Add New
+      </ButtonSecondary>
+    </Flex>
+  );
+}
+
+function AdminRule({
+  value,
+  isProcessing,
+  validation,
+  onChange,
+  onRemove,
+}: SectionProps<RuleModel, AdminRuleValidationResult> & {
+  onRemove?(): void;
+}) {
+  const { resources, verbs } = value;
+  const theme = useTheme();
+  return (
+    <Section
+      title="Admin Rule"
+      tooltip="A rule that gives users administrative access to certain kinds of resources"
```

Comment:
Feedback for the design choice, I don't think it's blocking for this PR: as a Teleport user, I'm kind of surprised this is called "admin rules". I don't think our documentation uses such concept and I wouldn't really consider them to be exclusively about administrative access: certain verbs such as read or list are needed just to use resources, not administer them.

Reply:
@ravicious @zmb3 Any better ideas, then?

Reply:
@gzdunek Yes, but this is an umbrella term for _all_ of the data in this form.

Reply:
(Sorry, misread your comment. Yes, you're right.)